Makefile

```diff
@@ -10,9 +10,15 @@ DOCKER_CACHE       := $(CURDIR)/.cache
 ANTCTL_BINARY_NAME ?= antctl
 OVS_VERSION        := $(shell head -n 1 build/images/deps/ovs-version)
 GO_VERSION         := $(shell head -n 1 build/images/deps/go-version)
+CNI_BINARIES_VERSION := $(shell head -n 1 build/images/deps/cni-binaries-version)
+NANOSERVER_VERSION := $(shell head -n 1 build/images/deps/nanoserver-version)
+WIN_BUILD_TAG      := $(shell md5sum build/images/base-windows/image-tag| head -c 32)
```

Comment:
Is it possible to use this format to generate the hash code. Then you don't need a single file to record a duplicate versions, which means two modifications as long as one version is changed.
```
echo "$GO_VERSION $CNI_BINARIES_VERSION $NANOSERVER_VERSION" | md5sum
```

Reply:
+1

Reply:
Updated